courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/1_core_tensorflow.ipynb

```diff
@@ -106,13 +106,21 @@
     "x = tf.Variable(2.0, dtype=tf.float32, name='my_variable')"
```

Comment:
<blockquote>the difference of the constants&nbsp;<code>a</code>&nbsp;and&nbsp;<code>b</code></blockquote><p>the product of the ...</p><p>Since it is a multiply</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/704/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>ah. good catch. will fix. </p>